packages/yoga/src/Card/native/EventCard/EventCard.jsx

```diff
@@ -0,0 +1,158 @@
+import React from 'react';
+import styled from 'styled-components';
+import { oneOf, shape, string } from 'prop-types';
+import { View } from 'react-native';
+import { Clock } from '@gympass/yoga-icons';
+
+import Card from '../Card';
+import Text from '../../../Text';
+
+const Event = styled(Card)`
+  flex-direction: row;
+
+  width: 280px;
+  height: 104px;
+  padding: 0;
+`;
+
+const DateInfo = styled(View)`
+  ${({
+    variant,
```

Comment:
```suggestion
    variant = EventCard.defaultProps.variant,
```

Since [EventCard has a `variant` prop](https://github.com/Gympass/yoga/pull/82/files#diff-0d8194769c9c5f57dbd53fb3449fba79R153), what do you think to set it as a default value for `<DateInfo />` `variant` prop? If you like this approach, you can remove the default value you do [here](https://github.com/Gympass/yoga/pull/82/files#diff-0d8194769c9c5f57dbd53fb3449fba79R26)

Reply:
This approach will not be necessary at moment because `DateInfo` will always receive the EventCard variant default.